Packs/Zoom/Integrations/Zoom/Zoom_test.py

```diff
@@ -935,3 +967,676 @@ def test_zoom_fetch_recording_command__not_able_to_delete(mocker):
+    """
+    manual_list_user_channel_pagination_mock = mocker.patch.object(Zoom, "manual_list_user_channel_pagination")
+    mocker.patch.object(Client, "generate_oauth_token")
+    client = Client(base_url='https://test.com', account_id="mockaccount",
+                    client_id="mockclient", client_secret="mocksecret")
+    from Zoom import zoom_list_user_channels_command
+    zoom_list_user_channels_command(client=client, user_id="user_id", limit=5)
+    assert manual_list_user_channel_pagination_mock.called
+
+
+def test_zoom_create_channel_command(mocker):
+    """
+    Given -
+        client
+    When -
+        creating a Zoom channel
+    Then -
+        Validate that the zoom_create_channel function is called with the correct arguments
+        Validate the command results including outputs and readable output
+    """
+    # Mock the response from zoom_create_channel
+    response = {
+        "id": "channel_id",
+        "name": "TestChannel3",
+        "type": "1",
+        "channel_url": "https://zoom.us/channel/channel_id"
+    }
+    zoom_create_channel_mock = mocker.patch.object(Client, "zoom_create_channel", return_value=response)
+
+    mocker.patch.object(Client, "generate_oauth_token")
+    client = Client(base_url='https://test.com', account_id="mockaccount",
+                    client_id="mockclient", client_secret="mocksecret")
+    from Zoom import zoom_create_channel_command
+
+    # Set up the inputs
+    user_id = "u12345"
+    member_emails = ["user2@example.com", "user1@example.com"]
+    add_member_permissions = "All channel members can add"
+    posting_permissions = "All members can post"
+    new_members_can_see_prev_msgs = True
+    channel_name = "TestChannel"
+    channel_type = "Private channel"
+
+    zoom_create_channel_command(client=client,
+                                user_id=user_id,
+                                member_emails=member_emails,
+                                add_member_permissions=add_member_permissions,
+                                posting_permissions=posting_permissions,
+                                new_members_can_see_prev_msgs=new_members_can_see_prev_msgs,
+                                channel_name=channel_name,
+                                channel_type=channel_type)
+
+    # Verify the API call
+    expected_url_suffix = f"/chat/users/{user_id}/channels"
+    expected_json_data = {
+        "channel_settings": {
+            "add_member_permissions": 1,
+            "new_members_can_see_previous_messages_files": True,
+            "posting_permissions": 1,
+        },
+        "members": [
+            {"email": "user2@example.com"},
+            {"email": "user1@example.com"}
+        ],
+        "name": "TestChannel",
+        "type": 1
+    }
+    zoom_create_channel_mock.assert_called_with(expected_url_suffix, expected_json_data)
+
+
+def test_zoom_delete_channel_command(mocker):
```

Comment:
please add doctoring for every set that explain waht is being run and what is validated in the end.
there is documentation in there tests, but in many it is missing.

Reply:
DONE